DESCRIPTION

```diff
@@ -1,15 +1,52 @@
 Package: geoGraph
-Version: 1.0-0
-Date: 2015/09/13
-Title: walking through the geographic space using graphs
-Author: Thibaut Jombart
+Type: Package
+Title: What the Package Does (Title Case)
```

Comment:
Maybe use the current title (or another more appropriate one?)

Reply:
Yes, that would make sense...